snowpack/src/build/build-pipeline.ts

```diff
@@ -122,7 +122,7 @@ async function runPipelineTransformStep(
     try {
       for (const destExt of Object.keys(output)) {
         const destBuildFile = output[destExt];
-        const {code} = typeof destBuildFile === 'string' ? {code: destBuildFile} : destBuildFile;
+        const {code} = destBuildFile;
```

Comment:
the `destBuildFile` must be `{ code: string | Buffer, map: string? }` type after load step.